build-libuv.bash

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+#
+#  Assuming the proper development tools are in place, make the
+#  libraries needed, and produce a shared object suitable for loading
+#  the code for Pair Based Curves (PBC).
```

Comment:
Small nit: comment refers to PBC curves, but this is the libuv build script.

Reply:
Oh, thanks!